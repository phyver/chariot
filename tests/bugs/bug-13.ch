:type f x x x x x
