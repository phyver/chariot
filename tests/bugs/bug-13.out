>> option verbose is set to 1
>> f¹ x² x³ x⁴ x⁵ x⁶
>> with:
>>   - 1: 'a → 'a → 'a → 'a → 'a → 'l
>>   - 2: 'a
>>   - 3: 'a
>>   - 4: 'a
>>   - 5: 'a
>>   - 6: 'a
>> 
>> with free variables: f : 'a → 'a → 'a → 'a → 'a → 'l , x : 'a
